src/rpc/blockchain.cpp

```diff
@@ -613,15 +597,12 @@ static RPCHelpMan getmempoolancestors()
                     HelpExampleCli("getmempoolancestors", "\"mytxid\"")
             + HelpExampleRpc("getmempoolancestors", "\"mytxid\"")
                 },
-        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+        [](const RPCContext& ctx) -> UniValue
 {
-    bool fVerbose = false;
-    if (!request.params[1].isNull())
-        fVerbose = request.params[1].get_bool();
+    bool fVerbose = ctx.param<bool>(1).defaults(false);
+    uint256 hash = ctx.param<uint256>(0);
```

Comment:
Same as elsewhere; this is not an optional value, so it shouldn't silently pass. (I won't mention these anymore, but there are others.)